package.json

```diff
@@ -86,8 +87,7 @@
     "winston": "1.0.0"
   },
   "devDependencies": {
-    "@babel/runtime": "7.0.0-beta.46",
-    "auth0-extensions-cli": "^1.1.2",
+    "auth0-extensions-cli": "^4.0.4",
```

Comment:
We seemed to have removed babel dev dependency. Is it correct that it is removed because this dep is getting pulled from "auth0-extensions-cli"?

Reply:
babel/runtime is not a dev dependencies (not used during code generation or bundling). it is used at runtime by bundle.js.  The version of babel used is controlled by `auth0-extensions-cli` as the cli tool is dynamically building the babel and webpack config to process our files into a bundle.